microsoft-edge/webview2/get-started/win32.md

```diff
@@ -6,86 +6,375 @@ ms.author: msedgedevrel
+
+   This screenshot shows some WebView2 code (headers), that's already present in the file immediately after cloning (downloading) the repo.  Previously, that code was not added in the repo yet.  But now, all of the code for the present tutorial is already present in the repo.  This means that you don't actually need to do the copy/paste operations later in this tutorial.
+
+   To confirm that the header lines are already already present in the repo: in a different window or tab, see [WebView2Samples/ ... /GettingStartedGuides/Win32_GettingStarted/HelloWebView.cpp](https://github.com/MicrosoftEdge/WebView2Samples/blob/master/GettingStartedGuides/Win32_GettingStarted/HelloWebView.cpp).
+
+1. If it's not already present, paste the following code into `HelloWebView.cpp`:
+
+
+
+### Set the solution to use the Win10 SDK and Visual Studio toolset
+
+This step is only needed for older versions of Visual Studio, so it's likely you can skip it.  But you can take a look at this UI in any case:
+
+1. In Visual Studio's **Solution Explorer**, right-click the **WebView2GettingStarted** _project_ (not the solution with the same name), and then select **Properties**.
+
+1. Select **Configuration Properties** > **General**, and then (if it's not the correct setting already):
+
+   1. Modify **Windows SDK Version** to use the Win10 SDK.
+
+   1. Modify **Platform Toolset** to use a Visual Studio toolset.
+
+   These modifications are only necessary for older<!--define--> versions of Visual Studio.
+
+   Here's a Visual Studio 2017 screenshot showing some valid settings:
+
+   ![In Visual Studio 2017, set Windows SDK Version to 10, and Platform Toolset to Visual Studio.](../media/tool-version.png)
+
+   The following is a Visual Studio 2022 screenshot; the values were already correct, so no change was required:
+
+   ![In Visual Studio 2022, Windows SDK Version is already 10, and Platform Toolset is already Visual Studio.](../media/tool-version-2022.png)
+
+Visual Studio might display errors, because your project is missing the WebView2 header file.  The errors resolve after completing the next section, "Install the WebView2 SDK".
+
+Continue with the steps below.
 
 
 <!-- ====================================================================== -->
-## Step 1 - Create a single-window app
+## Step 4 - Install the Windows Implementation Library (WIL)
+
+
+### Learn about Windows Implementation Libraries (WIL)
+
+In a moment, below you'll install the following two tools, which make working with COM easier for this tutorial:
+
+*  **Windows Implementation Libraries (WIL)** - A header-only C++ library to make life easier for developers on Windows through readable, type-safe C++ interfaces for common Windows coding patterns.
+
+*  **Windows Runtime C++ Template Library (WRL)** - A template library that provides a low-level way to author and use Windows Runtime components.
```

Comment:
Is this actually installed?  No step mentions or shows it, and NuGet package shows "no dependencies".